styles/typography.css

```diff
@@ -16,3 +15,90 @@
   --font-family-serif: "Source Serif 4", serif;
   --bs-font-sans-serif: "Source Sans 3";
 }
+
+h1,
+h2,
+h3,
+h4,
+h5,
+h6,
+.h1,
+.h2,
+.h3,
+.h4,
+.h5,
+.h6 {
+  font-weight: 600;
+}
+
+h6,
+.h6 {
+  text-transform: uppercase;
+}
+
+dt {
+  font-weight: 600;
+}
+
+dd {
+  font-weight: 400;
+}
+
+p {
+  font-size: 1rem;
+  font-weight: 400;
+}
+
+/* xs */
+@media (width < 576px) {
+  p {
+    font-size: 0.9rem;
+  }
+}
+
+/* xl */
+@media (width >= 1200px) {
+  h1,
+  .h1 {
+    font-size: 2.25rem;
```

Comment:
This will cause a font-size jump when you cross the threshold.  Currently is scales smoothly to 2.5rem (` calc(1.375rem + 1.5vw);`) which is the default size at 1200px width. Now it jumps down.